controller:
  # fixed parameter
  DIAMETER: 0.064
  RADIUS: 0.133414
  STRRESOLUTION: 1280
  DRVRESOLUTION: 384 # [gear] from 15 to 24, 240 PPR
  # Steer Paramete
  MAX_Steer_PWM: 235 # 240くらいからオーバーシュートする
  STRKP: 140 # 150以上だと少しオーバーシュートする default=140
  STRKI: 0.01
  STRKD: 0.0
  STROFFSET: 10
  # Drive Parameter
  MAX_Drive_PWM: 80 # 現状はPMW 80くらいしか正常に動作しない
  DRVKP: 2.5
  DRVKI: 0.00
  DRVKD: 0.0
  DRVOFFSET: 0
  # Others
  FRIQUENCY: 100
  LIMIT: 0.25
  ACC: 5
  STEP: 10 # FF 実装のためのプログラム
  KF: 17.38
  KV: 1.5 # 4.0がmaxだが転倒しやすい
  KW: 2.5 # 2.5
